android/java/org/chromium/chrome/browser/onboarding/P3aOnboardingActivity.java

```diff
@@ -46,13 +47,23 @@ protected void onCreate(Bundle savedInstanceState) {
                         ? getResources().getString(R.string.p3a_onboarding_title_text_1)
                         : getResources().getString(R.string.p3a_onboarding_title_text_2));
         CheckBox p3aOnboardingCheckbox = findViewById(R.id.p3a_onboarding_checkbox);
-        p3aOnboardingCheckbox.setChecked(BravePrefServiceBridge.getInstance().getP3AEnabled());
+        boolean isP3aEnabled = true;
+        try {
+            isP3aEnabled = BravePrefServiceBridge.getInstance().getP3AEnabled();
+        } catch (Exception e) {
+            Log.e("P3aOnboarding", e.getMessage());
```

Comment:
let's do `isP3aEnabled = false` here, so it's disabled if failed

Reply:
@SergeyZhukovsky by default it should be enabled according to spec.

Reply:
User can still uncheck the selection from UI